x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/rest/RestDeleteInferenceEndpointAction.java

```diff
@@ -49,7 +55,15 @@ protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient
             taskType = TaskType.ANY;
         }
 
-        var request = new DeleteInferenceModelAction.Request(inferenceEntityId, taskType);
-        return channel -> client.execute(DeleteInferenceModelAction.INSTANCE, request, new RestToXContentListener<>(channel));
+        if (restRequest.hasParam(FORCE_DELETE_NAME)) {
```

Comment:
Does it work if we omit these `hasParam` calls? I'm hoping it'll just fallback to using the default if the parameter doesn't exist.

Reply:
seems to work!